ddl/serial_test.go

```diff
@@ -1158,6 +1159,60 @@ func (s *testSerialSuite) TestAutoRandom(c *C) {
 	})
 }
 
+func (s *testSerialSuite) TestAutoRandomChangeFromAutoInc(c *C) {
```

Comment:
We can not modify primary key with `KEY` keyword, not sure if it is reasonable:

```
tidb> create table t(a bigint key);                                                       Query OK, 0 rows affected (0.01 sec)

tidb> alter table t modify a bigint key not null;ERROR 8200 (HY000): can't change column constraint - 1
```
I just leave a comment here in case.

Reply:
MySQL 8.0 result:
```sql
mysql> create table t(a bigint key);
Query OK, 0 rows affected (0.03 sec)

mysql> alter table t modify a bigint key not null;
ERROR 1068 (42000): Multiple primary key defined
```